packages/web-app-external/src/App.vue

```diff
@@ -64,12 +63,13 @@ export default defineComponent({
   },
 
   data: () => ({
+    appUrl: '',
+    errorMessage: '',
+    formParameters: {},
     loading: false,
     loadingError: false,
-    errorMessage: '',
-    appUrl: '',
     method: '',
-    formParameters: {}
+    resource: <Resource>{}
```

Comment:
I figured we'd need the resource for the `AppTopBar` 😇  and since we do an API call in the `created()` hook anyways we might as well fetch the whole thing